cli/pcluster/easyconfig.py

```diff
@@ -75,28 +75,30 @@ def prompt(prompt, default_value=None, hidden=False, options=None, check_validit
 
 @handle_client_exception
 def get_regions():
-    ec2 = boto3.client("ec2")
-    regions = ec2.describe_regions().get("Regions")
-    return [region.get("RegionName") for region in regions if region.get("RegionName") not in unsupported_regions]
+    try:
+        ec2 = boto3.client("ec2")
+        regions = ec2.describe_regions().get("Regions")
+        return [region.get("RegionName") for region in regions if region.get("RegionName") not in unsupported_regions]
+    except NoRegionError:
```

Comment:
shall we move this to handle_client_exception? Also shall we improve the message we display in case of non-configured credentials?

Reply:
I've opted to remove the ability to set credentials in `pcluster configure` and remove them from the sample`config` file, while still remaining backward compatible and allowing them to be set in the `~/.parallelcluster/config` file.

Reply:
^ That was in response to:

> Shall we also get rid of __init_credentials from cfnconfig and cleanup all boto3 initializations where we explicitly set the credentials from the config object?

In response to:

> shall we move this to handle_client_exception? Also shall we improve the message we display in case of non-configured credentials?

Done ✅